src/dolphin/filtering.py

```diff
@@ -0,0 +1,196 @@
+import numpy as np
+from numpy.typing import ArrayLike
+from scipy import ndimage
+
+
+def filter_long_wavelength(
+    unwrapped_phase: ArrayLike,
+    correlation: ArrayLike,
+    mask_cutoff: float = 0.5,
+    wavelength_cutoff: float = 50 * 1e3,
+    pixel_spacing: float = 30,
+) -> np.ndarray:
+    """Filter out signals with spatial wavelength longer than a threshold.
+
+    Parameters
+    ----------
+    unwrapped_phase : np.ndarray, 2D complex array
+        Unwrapped interferogram phase to filter.
+       correlation : Arraylike, 2D
```

Comment:
```suggestion
    correlation : Arraylike, 2D
```
ah this one was just a formatting problem

Reply:
Oh I was wondering about that one. Thank you!